README.md

```diff
@@ -7,30 +7,32 @@ You will find here scripts allowing to very easily build Qt with OpenSSL support
 
 Scripts available for these Qt versions:
 
-* [5.15.1][5151]
+* [5.15.2][5152]
 * [4.8.7][487]
 
-[5151]: https://github.com/jcfr/qt-easy-build/tree/5.15.1#readme
+[5152]: https://github.com/jcfr/qt-easy-build/tree/5.15.2#readme
 [487]: https://github.com/jcfr/qt-easy-build/tree/4.8.7#readme
 
 | Qt Version   | Linux                                                   | macOS                                                   | Windows CI |
 |--------------|---------------------------------------------------------|---------------------------------------------------------|------------|
-| 5.15.1       | [![Build Status][5151_linux_i_azure]][5151_linux_azure] | [![Build Status][5151_macos_i_azure]][5151_macos_azure] | [![Build Status][5151_windows_i_azure]][5151_windows_azure]         |
+| 5.15.2       | [![Build Status][5152_linux_i_azure]][5152_linux_azure] | [![Build Status][5152_macos_i_azure]][5152_macos_azure] | [![Build Status][5152_windows_i_azure]][5152_windows_azure]         |
 | 4.8.7        | NA                                                      | NA                                                      | NA         |
 
-[5151_linux_azure]: https://dev.azure.com/jamesobutler/qt-easy-build/_build/latest?definitionId=1&branchName=5.15.1
-[5151_linux_i_azure]: https://dev.azure.com/jamesobutler/qt-easy-build/_apis/build/status/jamesobutler.qt-easy-build?branchName=5.15.1&jobName=Linux
+[5152_linux_azure]: https://dev.azure.com/jamesobutler/qt-easy-build/_build/latest?definitionId=1&branchName=5.15.2
```

Comment:
This should point to your azure qt-easy-build repo. For some reason the last build information can't be obtained. Did you manually delete it? https://dev.azure.com/jchrisfillionr/qt-easy-build/_build/results?buildId=2&jobId=019c090b-adc9-5d3d-bff9-f0ed5003c51c

Reply:
>Did you manually delete it?

I don't think so. The only build there is this one:

![image](https://user-images.githubusercontent.com/219043/114951838-22cf7f80-9e23-11eb-8162-23a425b9c077.png)

Reply:
Ok I was able to get a build running https://dev.azure.com/jchrisfillionr/qt-easy-build/_build/results?buildId=3&view=results as part of #64.